openedx/core/djangoapps/content/course_overviews/migrations/0010_auto_20160329_2317.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('course_overviews', '0009_readd_facebook_url'),
+    ]
+
+    operations = [
+        migrations.RemoveField(
+            model_name='courseoverview',
+            name='facebook_url',
+        ),
```

Comment:
I don't see the code change associated with this operation. Should it have been committed previously?